### DISCUSSION TITLE
Using the online cron to updating feeds failed

### DISCUSSION THREAD

**Discussion by lga2011** [FreshRSS/FreshRSS#6054] [Q&A] [answered]
freshrss version: 1.23.1

online cron website:
https://cron-job.org
https://www.easycron.com

update feed url:
https://mywebsite/i/?c=feed&a=actualize

online cron response:
"This site requires Javascript to work, please enable Javascript in your browser or use a browser with Javascript support"

**Comment by Alkarex** [comment, marked as answer]
Please check the doc: https://freshrss.github.io/FreshRSS/en/users/09_refreshing_feeds.html#online-cron
The link is clearly wrong as it does not contain the `token` for access control.
If you check the output of the link you are trying, it is likely the login form.

**Comment by Rebelion** [comment]
I have the same problem as CJN1986, but I have followed the instructions on the doc [URL](https://freshrss.github.io/FreshRSS/en/users/09_refreshing_feeds.html#online-cron). However, on cron-job.org, it still gives the same output:

My update feed is :
http://mydomain.com/FrRSS/p/i/?c=feed&a=actualize&user=myuser&token=mytoken&ajax=1

**200 Success**,

"This site requires Javascript to work, please enable Javascript in your browser or use a browser with Javascript support."

```
<html>
  <body>
    <script type="text/javascript" src="/aes.js"></script>
    <script>
      function toNumbers(d) {
        var e = [];
        d.replace(/(..)/g, function(d) {
          e.push(parseInt(d, 16))
        });
        return e
      }

      function toHex() {
        for (var d = [], d = 1 == arguments.length && arguments[0].constructor == Array ? arguments[0] : arguments, e = "", f = 0; f < d.length; f++) e += (16 > d[f] ? "0" : "") + d[f].toString(16);
        return e.toLowerCase()
      }
      var a = toNumbers("f655ba9d09a112d4968c63579db590b4"),
        b = toNumbers("98344c2eee86c3994890592585b49f80"),
        c = toNumbers("83c897298529f040ab0cb442062c89b3");
      document.cookie = "__test=" + toHex(slowAES.decrypt(c, 2, a, b)) + "; expires=Thu, 31-Dec-37 23:55:55 GMT; path=/";
      location.href = "http://mydomain.com/FrRSS/p/i/?c=feed&a=actualize&user=myuser&token=mytoken&ajax=1&i=1";
    </script>
    <noscript>This site requires Javascript to work, please enable Javascript in your browser or use a browser with Javascript support</noscript>
  </body>
</html>
```
When I follow this url ( http://mydomain.com/FrRSS/i/?c=feed&a=actualize&user=myuser&token=mytoken&ajax=1 ) from doc, it's give me 404 not found, and the feed is not updated.

So I modified into this url ( http://mydomain.com/FrRSS/p/i/?c=feed&a=actualize&user=myuser&token=mytoken&ajax=1 ), it's give me "OK", but the feed still not updated.

**Reply by Alkarex** [reply]
> it's give me "OK", but the feed still not updated

Remember that not all feeds are updated every time: each feed has a max refresh rate. See also the `maxFeeds` parameter https://freshrss.github.io/FreshRSS/en/users/09_refreshing_feeds.html#online-cron
Just try again in a couple of hours to see whether there is any difference.
Check your FreshRSS logs + syslog to find out what happens more precisely.
